README.md

```diff
@@ -15,6 +15,12 @@ $ npm install
 $ sed -i 's/mostPackage/my-awesome-package/g' package.json
 ```
 
+or one-line install:
+
+```
+bash <(curl -sSL https://git.io/vr1HI)
```

Comment:
yeah i think if do that instead of process substitution, it wont pause to get the user's input with `read` :(